core/src/main/java/hudson/model/TopLevelItemDescriptor.java

```diff
@@ -135,10 +135,10 @@ public String getDisplayName() {
     }
 
     /**
-     * A description of this kind of item type. This description can contain HTML code but it is recommend to use text plain
-     * in order to avoid how it should be represented.
+     * A description of this kind of item type. This description can contain HTML code but it is recommended that
+     * you use plain text in order to be consistent with the rest of Jenkins.
      *
-     * This method should be called in a thread where Stapler is associated, but it will return an empty string.
+     * This method should be called in a thread where a Stapler is associated, otherwise it will return an empty string.
```

Comment:
Stapler is the Jenkins web framework, so `a Stapler` doesn't really work. It's not the office tool.

Perhaps *from a thread handling an HTTP request*? Although I'm not 100% certain these are equivalent.

Reply:
How about _from a thread where Stapler is handling an HTTP request_? That _sounds_ like what it's trying to say. Admittedly, we could be wrong in understanding how it behaves at edges, but...

Reply:
seems ok.